loopy/program.py

```diff
@@ -46,7 +46,7 @@
 
```

Comment:
Rename this file? (Nobody outside loopy should be importing this directly anyhow.)

Reply:
b95a2dc